### ISSUE TITLE
Memory efficiency of Send() with large file to many clients.

### ISSUE THREAD

**Issue by Altiano** [Eastrall/Ether.Network#112]
In my understanding,

`NetPacket` is buffered up with content and then send it with the call of `NetUser.Send()`

Question is,
What if I need to send large file (~100 MB) to many clients (~50) at the same time.?

If it is indeed buffered up before get send to the client.
The best way that I come up with is to split the data into (~1 MB) chunks and send it to the client 100 times.

Like this:

```
for index as integer = 1 to chunks_count

  dim chunk as byte() = GetChunk(data, index)
  using p as new NetPacket()
    p.write(chunk)
    NetUser.Send(p)
  end using

end for
```

**Comment by Eastrall** [comment]
Indeed, `NetPacket` is buffered. Ether has not been designed for to send large files throught the network.

I'll study the case and see if we can add this behavior easily. Thanks for reporting the issue.
